numpy/core/src/umath/loops_comparison.dispatch.c.src

```diff
@@ -0,0 +1,451 @@
+/*@targets
+ ** $maxopt baseline
+ ** sse2 sse41 avx2 avx512f avx512_skx
```

Comment:
```suggestion
 ** sse2 sse42 avx2 avx512f avx512_skx
```
targets `sse41` just for `_mm_cmpeq_epi64` is kinda excess regards to cost of binary size, maybe `sse42` would be better choice at least it comes with `_mm_cmpgt_epi64`.

Reply:
Done!